app/models/automation_task.rb

```diff
@@ -1,6 +1,6 @@
 class AutomationTask < MiqRequestTask
   alias_attribute :automation_request, :miq_request
-
+  has_one :container_deployment
```

Comment:
@alongoldboim can you give a quick description of what this relationship means?

I just [commented below](https://github.com/ManageIQ/manageiq/pull/7779/files#r60086923) regarding the relationship between `VM`s and `container_deployments`.  And, I'm not sure if this is a similar concept.  Or, something different.

If I had to guess, I'd think this means that an `AutomationTask` can result in a `ContainerDeployment` and you want to preserve that linkage.

It almost feels better to subclass `AutomationTask` under `ManageIQ::Providers::ContainerManager` and move this relationship there.  That way, only `AutomationTask`s from a container manager would result in a linkage like this.  But, that might be the wrong assumption.